tests/integration/data/test_int_element_data.py

```diff
@@ -1,9 +1,43 @@
-import footbot.data.element_data as ed
+import os
+from pathlib import Path
+
+import pandas as pd
+import pytest
+
+from footbot.data.element_data import get_bootstrap
+from footbot.data.utils import run_query
+from footbot.data.utils import set_up_bigquery
+from footbot.data.utils import write_to_table
 
 
 def test_get_bootstrap():
-    data = ed.get_bootstrap()
+    data = get_bootstrap()
     assert "elements" in data
     assert "events" in data
     assert isinstance(data["elements"], list)
     assert isinstance(data["events"], list)
+
+
+def test_write_element_data_to_table():
+    secrets_path = os.path.join(
+        Path(__file__).parents[3], "secrets/service_account.json"
+    )
+
+    client = set_up_bigquery(secrets_path)
```

Comment:
if you move the `client` fixture from `test_int_simulator.py ` to something like `tests/integration/conftest.py` you can re-use it in all tests